src/tests/system/tests/test_memory_cache.py

```diff
@@ -1656,3 +1659,366 @@ def test_memory_cache__truncate__nosigbus(client: Client, ldap: LDAP):
     result = client.tools.id("user-1")
     assert result is not None, "User was not found"
     assert result.user.name == "user-1"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
```

Comment:
This runs the test against everything - LDAP, IPA, Samba - right?

I don't it  makes a lot of sense for tests that verify caching (especially memory cache that is between sss_client and a responder).